docs/source/guides/tools_menu.rst

```diff
@@ -196,6 +196,14 @@ Events Run On
 
 This function is also used to select the core that runs the Arduino events. This is only valid if the target SoC has 2 cores.
 
+Erase Flash
+***********
+
+This option selects the flash memory region to be erased before uploading the new sketch.
+
+* **Only Sketch** - Upload the sketch without erasing all flash contents. (default)
+* **All Flash Contents** - Erase all flash contents before uploading the sketch.
+
```

Comment:
If changes are accepted as above, the text in the documentation shall reflect it.